### PR TITLE
Intercoms have some nice overlays for indicating mic and broadcast

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
Adds some nice overlays for displaying whether the mic or broadcasting settings are enabled

## Why It's Good For The Game
Adds a purpose to the weird red / green light already on the sprite, makes it a bit easier to tell if someone (the ai) is spying on a conversation.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image
image
image

</details>

## Changelog
:cl:
tweak: intercom overlay sprites
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
This is good, finally a way to observe when an AI is spying on your conversations to instantly report you to sec without having to open the intercoms first

**Comment by etherware-novice** [comment]
ty!!

File: code/game/objects/items/devices/radio/intercom.dm [hunk 1]
```diff
@@ -20,6 +20,7 @@
 	if(!current_area)
 		return
 	RegisterSignal(current_area, COMSIG_AREA_POWER_CHANGE, .proc/AreaPowerCheck)
+	update_icon()
```
Comment (Penwin0) [inline code/game/objects/items/devices/radio/intercom.dm]: This is not necessary

Comment (etherware-novice) [reply code/game/objects/items/devices/radio/intercom.dm]: its just incase the intercom starts with one or the other on so it always has the right lights on

Comment (Penwin0) [reply code/game/objects/items/devices/radio/intercom.dm]: It'll do it anyways even without this proc here, try it yourself if you don't believe me

Comment (etherware-novice) [reply code/game/objects/items/devices/radio/intercom.dm]: kk

**Review by Penwin0 — APPROVED** [review]

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**